code/config/otel.yml

```diff
@@ -2,14 +2,20 @@ receivers:
   otlp:
     protocols:
       grpc:
+  prometheus:
+    trim_metric_suffixes: true
+    use_start_time_metric: true
+    start_time_metric_regex: .*
+    config:
+      scrape_configs:
+        - job_name: 'otel-collector-metrics'
+          scrape_interval: 5s
+          static_configs:
+            - targets: ['127.0.0.1:8888']
 
 exporters:
   debug:
     verbosity: detailed
-  otlp/dashboard:
```

Comment:
This needs to be added back if we also want the Aspire dashboard to represent the otel information.